packages/glab/glab.nuspec

```
@@ -26,4 +26,4 @@
       <dependency id="glab.portable" />

```

```suggestion
      <dependency id="glab.portable" version="[1.24.1]" />
```

I saw recently a suggestion to be explicit with the dependency version numbers to ensure the proper packages are being installed.